src/app/widgets/buses/admin-panel-buses/admin-panel-buses.component.html

```diff
@@ -0,0 +1,36 @@
+<div class="container">
+  <div class="row">
+    <div>
+      <form [formGroup]="defaultBusStopForm" (ngSubmit)="(onSubmit)">
+        <div id="user-data">
+          <div class="form-group">
+            <label for="defaultBusStop" class="defaultBusStop"
+              >Please, choose your <br />default bus stop from the list:</label
+            >
+            <select
+              class="form-control"
+              formControlName="defaultBusStop"
+              size="5"
+            >
+              <option
+                *ngFor="let busStop of busStops | onlyuppercase | norepeats"
+                [value]="busStop.id"
+              >
+                {{ busStop.name | titlecase }}
+              </option>
+            </select>
+          </div>
+        </div>
+        <button
+          type="button"
```

Comment:
(click)="onSubmit()" nie jest potrzebny bo to jest formularz. Tylko aby zadziałało potrzeba kilka zmian
1. wyrzuć type="button" bo powinien być type="submit" (to jet domyślny typ w elemencie form
2. wyrzuć (click)="onSubmit()"
3. w <form zamień (ngSubmit)="(onSubmit)" na (ngSubmit)="onSubmit()"